bin/terminus

```diff
@@ -16,15 +16,8 @@ if (version_compare(PHP_VERSION, '7.4.0', '<') === true) {
     echo 'Upgrade to PHP 7.4 or newer to use Terminus 3. For PHP versions prior to 7.4, downgrade to Terminus 2.x.'; echo "\n\n";
     exit(1);
 } elseif (version_compare(PHP_VERSION, '8.1.0', '>=') === true) {
-    echo "\n";
-    echo 'PHP 8.1+ is not supported by this version of Terminus.' . "\n";
-    echo 'Check for new versions at https://github.com/pantheon-systems/terminus/releases' . "\n";
-    echo "\n";
-    if (!getenv('TERMINUS_ALLOW_UNSUPPORTED_NEWER_PHP')) {
-        echo 'Set environment variable TERMINUS_ALLOW_UNSUPPORTED_NEWER_PHP to try continuing anyway.' . "\n";
-        echo "Stopping.\n\n";
-        exit(1);
-    }
+    // Prevent outputting PHP deprecation messages produced by vendors.
+    error_reporting(error_reporting() & ~E_DEPRECATED);
```

Comment:
Sure, I can go with the pragmatic solution. We should still fix our dependencies and remove this as follow-on work.